### ISSUE TITLE
[Feature request] Ability to still use `url-loader` for certain SVGs

### ISSUE THREAD

**Issue by dustinhorton** [zabute/gatsby-plugin-svgr#3]
I'd love to be able to enable this plugin, but still be able to use SVG files in Sass/CSS and JSX. I don't think that's currently possible?

I'm not really literate in Webpack (especially v1), but it seems like v3's [`oneOf`](https://webpack.js.org/configuration/module/#rule-oneof) combined with  a [`resourceQuery`](https://webpack.js.org/configuration/module/#rule-resourcequery) would make this possible…but again, not sure how to do that in v1.

I was picturing something like this:

```jsx
import LogoSvg from 'svgs/logo.svg?component  // handled by SVGR
import logo from 'images/logo.svg  // handled by url-loader

[…]

<LogoSvg />
<img src={logo} />
```

…or maybe just a configureable path:

```jsx
import LogoSvg from 'svgs/logo.svg  // svg/**/*.svg handled by SVGR
import logo from 'images/logo.svg  // all others handled by url-loader

[…]

<LogoSvg />
<img src={logo} />
```

If this is already actually possible, I'd love some tips on how to configure it to work this way if you have some time. Thanks!

**Comment by zabute** [comment]
You can configure the loader right in your imports:
```jsx
import logo from '!url!../svgs/logo.svg'

<img src={logo}/>
```
But its hacky and gets uglier if you have queries. Your second suggestion is, I think, better since gatsby already exposes its api via the file system (layouts, pages), I don't see why this plugin shouldn't do the same.

How about  a ```dir: string``` key in the options which whitelists a directory to be processed by this plugin?

**Comment by dustinhorton** [comment]
That'd be awesome.

**Comment by zabute** [comment]
published in ```0.2.0```
